incubator/hnc/pkg/reconcilers/hierarchical_namespace_test.go

```diff
@@ -0,0 +1,94 @@
+package reconcilers_test
+
+import (
+	"context"
+
+	api "github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/api/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("Hierarchy", func() {
+	ctx := context.Background()
+
+	var (
+		fooName string
+		barName string
+	)
+
+	BeforeEach(func() {
+		fooName = createNS(ctx, "foo")
+		barName = createNSName("bar")
+
+		// Create "bar" hns in "foo" namespace
+		foo_hns_bar := newHierarchicalNamespace(barName, fooName)
+		updateHierarchicalNamespace(ctx, foo_hns_bar)
+	})
+
+	It("should set the self-serve subnamespace as a child on the current namespace", func() {
+		if !enableHNSReconciler {
```

Comment:
Add in the `BeforeEach` instead?

Reply:
Good catch! I initially put them in the `BeforeEach` and it didnt early exit with `return`. After another research, I am now able to use `Skip()`. Thanks!

Reply:
Cool!